src/measure_theory/measurable_space.lean

```diff
@@ -205,6 +205,18 @@ end
 lemma measurable_of_finite [finite α] [measurable_singleton_class α] (f : α → β) : measurable f :=
 λ s hs, (f ⁻¹' s).to_finite.measurable_set
 
+lemma measurable_of_countable [countable α] [measurable_singleton_class α] (f : α → β) :
+  measurable f :=
+begin
+  intros s hs,
+  classical,
+  have : f ⁻¹' s = ⋃ x : α, ite (f x ∈ s) ({x} : set α) ∅,
+  { ext1 x, simp, },
+  rw this,
+  refine measurable_set.Union (λ x, _),
+  exact measurable_set.ite' (λ _, measurable_set_singleton x) (λ _, measurable_set.empty),
+end
```

Comment:
Here is a 1-line proof that doesn't need your new lemmas in `data.set.basic`. I didn't test it but it should work.
```suggestion
λ s hs, s.to_countable.measurable_set
```

Reply:
Thanks! I didn't realize that we had the set lemma. I should have looked at the lemma above (`measurable_of_finite`), which is also proved like this.